central/notifiers/syslog/syslog_test.go

```diff
@@ -104,6 +139,45 @@ func (s *SyslogNotifierTestSuite) TestCEFExtensionFromPairs() {
 	s.Equal(fmt.Sprintf("%s %s", extensionPair1, extensionPair2), extension)
 }
 
+func (s *SyslogNotifierTestSuite) TestValidateSyslogEmptyExtrafields() {
+
```

Comment:
[nit] extraneous newline (for all the methods below)

Reply:
done